schemas/rule.yml

```diff
@@ -0,0 +1,83 @@
+---
+- name: rule
+  title: Rule
+  group: 2
+  short: Fields to capture details about rules used by observers and agents to trigger events and/or actions.
+  description: >
+   Rule fields are used to capture the specifics of any observer or agent rules that are associated with a given action or event.
```

Comment:
can we change "are associated with a given action or event." to "generate alerts or other events."

Reply:
done